framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1449,6 +1569,12 @@ public boolean useUncheckedCodeDefault(String kindOfCode) {
         return false;
     }
 
+    /**
+     * Elements with a {@code @SuppressWarnings} that actually suppressed a warning for this
+     * checker.
+     */
+    public final Set<Element> elementsWithSuppressedWarnings = new HashSet<>();
```

Comment:
Can this field be `protected` now? Should the documentation make it more explicit that usually nobody but the framework needs to modify it?